### ISSUE TITLE
Multiplayer Suggestions

### ISSUE THREAD

**Issue by harrysbird** [OpenRCT2/OpenRCT2#2897]
**I Have Suggestions.**

**1. We Should add Server destruction on a multiplayer server list that can see like in openttd, you can change in server settings.

2. Add Kick Reason before kicking player, because i don't want to kick player for no reason.

3. Add Language spoken in server settings

4. Add Server Secruity Type in server settings:**
-  Anyone can join - Any Player can join your server
-  Invite Only - Admin need Accept before player joining your server
-  Password - Player need to type password before joining your server

**Comment by pizza2004** [comment]
What exactly is Server destruction?

**Comment by harrysbird** [comment]
it was wrong word i changed it to: description

**Comment by Gymnasiast** [comment]
Please don't put everything in bold next time.

**Comment by thyeggman** [comment]
6 & 7 are the same essentially, and password protection is already implemented.  5 is the default behavior, and 4 a little unclear... should there be more security than 'password protected'?

**Comment by Gymnasiast** [comment]
@thyeggman : Markdown messed up his message, causing three subpoints to appear as numbered points. I just fixed that.

**Comment by telk5093** [comment]
I agree with 1(description) and 3(server spoken lang) of him. I think it must be implemented as a new feature.
But I think 2 and 4 are unneccesary since server chats before ban and server security could be controlled by password.
